hedera-node/src/main/java/com/hedera/services/ServicesState.java

```diff
@@ -425,6 +406,8 @@ private void internalInit(
 				/* This was an upgrade, discard now-obsolete preparation state */
 				networkCtx().discardPreparedUpgradeMeta();
 				dualState.setFreezeTime(null);
+				log.info("START OVER START OVER START OVER");
```

Comment:
I just realized we need to increment the block number post-upgrade, because the in-progress block will have been implicitly flushed [here](https://github.com/hashgraph/hedera-services/blob/master/hedera-node/src/main/java/com/hedera/services/stream/RecordStreamManager.java#L228) via `multiStream.close()`.

Something like,
```
networkCtx().incrementBlockNo(); 
networkCtx().setFirstConsTimeOfCurrentBlock(null);
log.info("First post-upgrade block is #{}", networkCtx().getBlockNo());
```